gtsam.h

```diff
@@ -674,6 +674,7 @@ class Rot3 {
   double roll() const;
   double pitch() const;
   double yaw() const;
+  pair<gtsam::Unit3, double> axisAngle() const;
```

Comment:
What about defining: 

```
using AxisAngle = std::pair<gtsam::Unit3, double>;
``` 
?

I'm aware of it probably being a matter of taste, since both options have pros and cons...

Reply:
`AxisAngle` is already a function in this class, so it might make it confusing (if it compiles). I had similar thoughts too (e.g. a struct) but like you said, it's a matter of taste.